regulations/uitests/diff_test.py

```diff
@@ -29,6 +30,24 @@ def test_diffs(self):
         # drawer button should be active
         self.assertTrue('current' in drawer_button.get_attribute('class'))
 
+        # drawer should display regulation history
+        timeline_list_items = self.driver.find_elements_by_class_name('history-toc-list')
+        expected_timeline_versions = [
+            '2011-11111',
+            '2012-12121'
+        ]
+        for index, item in enumerate(timeline_list_items):
+            self.assertEqual(item.get_attribute('data-base-version'), expected_timeline_versions[index])
+
+        # each timeline entry is a link to the rule effective on that date
+        timeline_links = self.driver.find_elements_by_class_name('version-link')
+        expected_timeline_urls = [
+            self.test_url + '/1005-2/2011-11111',
+            self.test_url + '/1005-2/2012-12121'
+        ]
+        for index, link in enumerate(timeline_links):
```

Comment:
Same comment as above.

Reply:
Fixed in 6381fa7eba9dc31ad2da21ab50f01339ecd78574. I also updated a similar test in `search_test.py` to iterate over the expected output instead of the actual output.